### ISSUE TITLE
Opening Alfred through direct shortcut seems to disable shortcuts globally

### ISSUE THREAD

**Issue by anshukla** [agzam/spacehammer#32]
Repro steps:

cmd + space => Spacehammer => space => alfred
Shortcuts seem to work fine. For example, I can I can use cmd + w to close tabs in a browser.

I had also bound alfred previously to opt + spacebar per default instructions. However, if I use alfred directly without going through Spacehammer, cmd + w to close tabs stops working.

My guess is it's related to some app specific shortcuts, but I haven't had time to debug. Hammerspoon console doesn't print anything when I invoke Alfred directly and modal state remains idle.

**Comment by agzam** [comment]
It might be related to #31

**Comment by eccentric-j** [comment]
@agzam Just to be thorough, are you sure that's happening on the develop branch?

**Comment by anshukla** [comment]
Yep, I have the develop branch pulled on my fork.

**Comment by agzam** [comment]
I can't reproduce this exact issue with Alfred (maybe it happens, I just can't catch exact moment of it), but I am experiencing #31 fairly often (which is quite annoying).

**Comment by anshukla** [comment]
I investigated further. Key to my configuration which was causing the issue was that I also had Spotlight bound to opt+spacebar. As a result, when I would do opt+spacebar, Spotlight would briefly appear and then be replaced with Alfred.

I disabled all keyboard shortcuts for Spotlight, and now this issue doesn't reproduce any longer.

Next, I tried the same repro steps after quitting Hammerspoon (to narrow down whether this was macOS, Alfred, or Hammerspoon). It still reproduces. So I think we should close the issue -- it's some sort of weird interaction between Alfred and Spotlight. Sorry for the noise!
